### PR TITLE
Renamed custom to common

### PR DESCRIPTION
**Pull request by DanDevDan**
Renamed custom to common and made the neccessary changes in main.py

### CODE REVIEW

**Review by ItsDrike — APPROVED** [review]
Umm, you could've just pushed another commit fixing it to #44, no need for new pull request, but ok.

It looks good, but I'll wait for @janaSunrise to also review this before we merge

**Review by janaSunrise — APPROVED** [review]

**Pull request merged**
